README.md

```diff
@@ -1,108 +1,96 @@
 # __StonkScraper__
-This API is a WIP. Check back often for new features!<br>
 [See a live implementation here!](https://boringstocks.live/)
 
-`StonkScraper` is a lightweight API that scrapes Yahoo's financial pages for a defined stock index. It is minimal by design but returns most of the data that the average daytrader would require to decide whether to buy or sell GME.
+`StonkScraper` is a lightweight API built off the robin_stock API.
 
 ## Usage
+Primary endpoint: `http://api.boringstocks.live/v2`<br>
+Index data: HTTP GET  `https://api.boringstocks.live/v2/<stock_index>`<br>
+Historical data: HTTP GET `https://api.boringstocks.live/v2/<stock_index>/historical/<data_range>`
 
-Primary endpoint: `http://api.boringstocks.live/`
+### Index Data
+`StonkScraper` returns a JSON containing:
 
-### Current Index Data
-Request `https://api.boringstocks.live/v1/<stock_index>`<br>
-`StonkScraper` returns a JSON containing these data points:
-  - Current price
-  - Range of historical data (not including the current day)
-  - Market cap
-  - Market status (open/close)
-  - Opening price
-  - Price change (points & percent)
-  - Day's range (date, high, low, & close)
-  - Index
-  - Timestamp
-  - Average volume
-  - Volume
+Data | Key | Type
+---- | --- | ----
+Current price | `"current"` | float
+Market cap | `"market_cap"` | float
+Market status (closed/open) | `"market_status"` | int (0 & 1) 
+Index name | `"name"` | string
+Points/Percent change | `"points_change"` | dict
+Price ranges (date/high/low/open) | `"range"` | dict
+Index symbol | `"symbol"` | string
+Timestamp | `"timestamp"` | string
+Average volume | `"avg_volume"` | float
+Volume | `"volume"` | float
 
-### Historical Index Data
-`StonkScraper` can also return the historical data ending on the current date.<br>
-Request `https://api.boringstocks.live/v1/<stock_index>/historical/<data_range>`
-Desired Range | <data_range>
-------------- | ------------
-5 days | 5_days
-1 month | 1_month
-6 months | 6_months
-1 year | 1_year
-All historical data | max
 
-## Examples
-Requesting `https://api.boringstocks.live/v1/GOOG`:
+### Historical Data
+`StonkScraper` returns a JSON containing:
```

Comment:
I feel that this line is off. The table bellow is related to all the possible endpoints of this route, not what the JSON contains.